Readme.md

```diff
@@ -1,9 +1,40 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128579957/20.1.10%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T386418)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
+#  Dashboard for Web Forms - How to Load and Save Dashboards from/to a Database
 
+This example shows how to create a custom dashboard storage that allows you to store dashboards in a database.
+
+<!-- default file list -->
+## Files to Look At
+
+* [DataBaseEditaleDashboardStorage.cs](./CS/DataBaseEditaleDashboardStorage.cs) (VB: [DataBaseEditaleDashboardStorage.vb](./VB/DataBaseEditaleDashboardStorage.vb))
+* [Global.asax.cs](./CS/Global.asax.cs) (VB: [Global.asax.vb](./VB/Global.asax.vb))
+<!-- default file list end -->
+
+## Example Overview
+
+The example uses the [System.Data.SqlClient](https://docs.microsoft.com/en-us/dotnet/api/system.data.sqlclient?redirectedfrom=MSDN&view=net-5.0) members to connect and operate an MS SQL server database.
+
+A custom dashboard storage should implement one of the following interfaces: [IDashboardStorage](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWeb.IDashboardStorage) or [IEditableDashboardStorage](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWeb.IEditableDashboardStorage).
+
+The following API used in the example:
+
+- [LoadDashboard](https://docs.devexpress.com/Dashboard/DevExpress.DashboardWeb.IDashboardStorage.LoadDashboard(System.String)) 
+
+    Returns a dashboard by its ID in the XDocument format, which describes an object model of the dashboard.
```

Comment:
- `Returns a dashboard by its ID` Do you mean 'Returns a dashboard with the corresponding ID'?
- `in the XDocument format` -> in XDocument format
- `which describes` That does this refer to? The dashboard or the format? 'which' is covered in our guidelines

Reply:
Loads a dashboard with the specified ID in XDocument format from storage.

Reply:
```suggestion
    Loads a dashboard with the specified ID in XDocument format from storage.
```